config/environments/development.rb

```diff
@@ -56,12 +63,18 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
+  # Uncomment if you wish to allow Action Cable access from any origin.
+  # config.action_cable.disable_request_forgery_protection = true
+
```

Comment:
Do we need to add config.hosts << ".#{ENV['APP_ROOT_DOMAIN']}" here as well? Perhaps it should be in application.rb?

Reply:
I moved it to its own initializer file.